lib/source.js

```diff
@@ -15,10 +15,22 @@ const path = require('path');
 
 class Source {
     constructor(paramsSource, directory="", name="") {
+        this.sourceSettings = paramsSource;
```

Comment:
so to access this inside a worker callback, it would be:
```
source.sourceSettings
```

IMO, the `source` prefix is repetitive. also, remember now this could be a watermark asset and not a source file so we should probably keep generic and just call it `settings`
wdyt?

Reply:
or even just `params`

Reply:
we should probably change the name of this class to `Asset` since it is more general than source. (this can be done in a separate ticket)

Reply:
@jdelbick `sourceSettings` refers to the class name, which is `Source`...

Reply:
@tmathern Yes obviously the source prefix was referring to the name of the class...that was not in question.

Here are my thoughts:
1. this `Source` class creates a source object that we pass to the rendition callback. So inside the rendition callback, (workers), someone would access source parameters like this `source.sourceSettings` which is repetitive. We could drop the `source` and just call it settings. that was we access it like `source.settings`
2. we create `Source` objects when we download watermarks now: https://github.com/adobe/asset-compute-sdk/blob/master/lib/storage.js#L47 so I was suggesting we change the name of the class from`Source` -> `Asset` because it is more general (this can be done in a separate ticket)

Reply:
1. I already updated `sourceSettings` to `settings` :).
2. I would not do the name change here (aka create a ticket): because this removes backwards compatibility because of the name change. Which means updating every location using `Source` class. Also, it would also mean double-checking on core workers and custom workers that they didn't break (and their unit tests). Changing names like that is not a small feat anymore, since we GA'd.

Reply:
> Which means updating every location using Source class with the new name and access method (the getters are here only for backwards compatibility).

When we made breaking changes to the `Source` class in mob session, we realized it is only used in `storage.js` and `source.test.js` unit tests

changing the name of the class won't change the behavior in any of the rendition callbacks because we still refer to the class instance as `this.source`: https://github.com/adobe/asset-compute-sdk/blob/master/lib/worker.js#L169-L174

but I am fine leaving it as is, not too important as discussed over slack